browser/css/jsdialogs.css

```diff
@@ -1218,6 +1223,20 @@ input[type='number']:hover::-webkit-outer-spin-button {
 	background-color: var(--color-background-darker);
 }
 
+.ui-iconview-entry img {
+	vertical-align: middle;
+}
+
+[data-theme='dark'] .sidebar-panel #layoutpanel_icons .ui-iconview-entry img,
```

Comment:
below rules please move at the end of icon view section and comment where to find them